tests/settings/e2e/steps/currency-display-settings.js

```diff
@@ -0,0 +1,36 @@
+import { When, Then } from 'cucumber';
+
+const CONVERTED_CURRENCY_WALLET_DISPLAY_GENERAL = '//*[@class="WalletSummaryCurrency_currencyRate"]';
+const DISPLAY_BALANCE_OTHER_CURRENCY_TOGGLE_STATUS_OFF = '//*[@class="SimpleSwitch_switch SwitchOverrides_switch"]';
+const DISPLAY_BALANCE_OTHER_CURRENCY_TOGGLE_STATUS_ON = '//*[@class="NormalSwitch_component SimpleSwitch_root SwitchOverrides_root SimpleSwitch_checked SwitchOverrides_checked"]';
+const SELECT_CURRENCY_DROPDOWN = '//*[@label="Select currency"]';
+
+When(/^I open currency selection dropdown$/, function() {
+  return this.waitAndClick(SELECT_CURRENCY_DROPDOWN);
+});
+
+When(/^I select "([^"]*)" as the selected displayed currency$/, function(currency) {
+  return this.waitAndClick(`//*[contains(text(), "${currency}")]`);
+});
+
+Then(/^I should see "([^"]*)" displayed beside wallet balance$/, function(ticker) {
+  return this.client.waitForVisible(`//*[@class="WalletSummaryCurrency_currencyRate" and text()="${ticker}"]`);
+});
+
+When(/^I toggle the button (on|off) to change if I want to see my ada balance in other currency's$/, function(switchStatus) {
+  let selector = DISPLAY_BALANCE_OTHER_CURRENCY_TOGGLE_STATUS_ON;
```

Comment:
The ternary operator works well in this kind of situation, so you don't have to reassign a value to the variable:

```js
const selector = switchStatus === 'on'
  ? DISPLAY_BALANCE_OTHER_CURRENCY_TOGGLE_STATUS_OFF
  : DISPLAY_BALANCE_OTHER_CURRENCY_TOGGLE_STATUS_ON;
```

Reply:
Fixed 👍